spec/definitions/EnterpriseUpdateV1.yaml

```diff
@@ -0,0 +1,55 @@
+type: object
+properties:
+  name:
+    title: Name
+    type: string
+    maxLength: 255
+    minLength: 1
+  display_name:
+    title: Display name
+    type: string
+    maxLength: 50
+    minLength: 1
+    x-nullable: true
+  short_code:
```

Comment:
Should we allow to update short_code ?

Reply:
It is allowed in current api. Should it not be allowed ?

Reply:
IMO this should not be allowed. @abdulMuneerM @bmehendale

Reply:
It should not be allowed. @tarun91vas @ammar2168